lib/matplotlib/tests/test_axes.py

```diff
@@ -4031,7 +4031,7 @@ def test_psd_noise():
 
 
 @image_comparison(baseline_images=['csd_freqs'], remove_text=True,
-                  extensions=['png'])
+                  extensions=['png'], tol=0.002)
```

Comment:
This fix is for the macOS test and doesn't affect this PR.  Happy to remove it if the commits will be cleaner.

Reply:
Can you throw this in another PR? It would be good to get to the bottom of the cause (I'll investigate later today) before just adding a tolerance.